x/transferinject/types/expected_keepers.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	rollapptypes "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+)
+
+// BankKeeper defines the expected interface needed
+type BankKeeper interface {
+	GetDenomMetaData(ctx sdk.Context, denom string) (types.Metadata, bool)
+	HasDenomMetaData(ctx sdk.Context, denom string) bool
```

Comment:
it's a nit, but here's only one usage of Has
Could we just have `Get`?

Reply:
fine